src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -46,4 +46,9 @@ public User(Long id, String name, String surname) {
         this.name = name;
         this.surname = surname;
     }
+
+    public User(String name, String surname) {
+        this.name = name;
+        this.surname = surname;
+    }
```

Comment:
Dodałam konstruktor do encji User bez pola id, bo nie przechodziły testy dla tej encji, pytanie czy to tak zostawić czy nie powinnam była tego robić?

Reply:
Konstruktor bez pola id ma sens, ponieważ ID powinno być nadawane przy zapisie do bazy